app/src/App.tsx

```diff
@@ -35,6 +35,11 @@ const RootQuery = graphql`
         startTime
         endTime
       }
+      corpusDataset {
+        name
+        startTime
+        endTime
```

Comment:
Actually, this goes to a context used by the point cloud display settings and cannot be trivially removed, i.e. it would require breaking parity with the reference dataset. So I'm going to keep it for now just to get the PR merged.